tests_spec/pfm.md

```diff
@@ -5,6 +5,6 @@
 | 1  | Rollapp to 3rd party chain  | | | <ul> <li> Try to send RAX from rollapp to an external chain, eibc should be switched off </li> <li> RAX should be available on a 3rd party chain  after rollapp grace period is reached </li> <li> no packet commitment left on the rollapp (i.e ack recieved). can be queries using the cli rollapp-evm q ibc channel packet-commitments transfer <channel-id> </li> </ul> | Partly solved <br> (lack query `packet commitment` left on the rollapp) | [test_pfm_with_grace_period](../tests/ibc_pfm_with_grace_period_test.go) |
 | 2  | 3rd party chain to Rollapp | | | <ul> <li> transfer from 3rd party chain to rollapp</li> <li> funds should be available on rollapp almost immediately </li> </ul> | 🛑 <br> (missing) | TODO |
 | 3  | Rollapp to 3rd party chain with eIBC | | | <ul> <li> send PFM + eIBC memo should fail with ackError on hub  </li> <li> no packet commitment left on the rollapp (i.e ack recieved). can be queries using the cli rollapp-evm q ibc channel packet-commitments transfer <channel-id> </li> </ul>| 🛑 <br> (missing) | TODO |
-| 4  | Rollapp to Rollapp - ERC20 registered | erc20  registered on rollapp2 for rollapp 1 IBC | 🛑 <br> (missing) | <ul> <li> send from rollapp 1 to rollapp 2 using PFM </li> <li> rollapp token 1 should be available on the rollapp 2  after rollapp1 grace period is reached </li> <li> no packet commitment left on the rollapp (i.e ack recieved). can be queries using the cli rollapp-evm q ibc channel packet-commitments transfer <channel-id> </li> </ul> | 🛑 <br> (missing) | TODO |
+| 4  | Rollapp to Rollapp - ERC20 registered | erc20  registered on rollapp2 for rollapp 1 IBC | ✅ | <ul> <li> send from rollapp 1 to rollapp 2 using PFM </li> <li> rollapp token 1 should be available on the rollapp 2  after rollapp1 grace period is reached </li> <li> no packet commitment left on the rollapp (i.e ack recieved). can be queries using the cli rollapp-evm q ibc channel packet-commitments transfer <channel-id> </li> </ul> | ✅ | [TestIBCPFM_RollApp1ToRollApp2WithErc20](https://github.com/dymensionxyz/e2e-tests/blob/0e2d1f9070c6a88b766b86ac94d93ab58de15cb4/tests/ibc_pfm_with_grace_period_test.go#L710)|
```

Comment:
This one should be relative link ../tests/ibc_pfm_with_grace_period_test.go#L710

Reply:
Updated